frontend/ui/widget/doublespinwidget.lua

```diff
@@ -23,29 +23,30 @@ local WidgetContainer = require("ui/widget/container/widgetcontainer")
 local _ = require("gettext")
 local Screen = Device.screen
 
-local HyphenationLimitsWidget = InputContainer:new{
-    title_text = _("Hyphenation limits"),
+local DoubleSpinWidget = InputContainer:new{
+    title_text = "",
     title_face = Font:getFace("x_smalltfont"),
+    info_text = "",
     width = nil,
     height = nil,
-    -- Min (2) and max (10) values are enforced by crengine
     left_min = 1,
     left_max = 10,
     left_value = 2,
```

Comment:
May be put back these values (and the comment) in readerhyphenation.lua.
And here, set defaults similar to the ones of (single)spinwidget, for coherency (even if no other widget will use these defaults).